test-app/build-tools/android-metadata-generator/src/src/com/telerik/metadata/Writer.java

```diff
@@ -61,7 +62,7 @@ private static void writeInt(int value, StreamWriter out) throws Exception {
     private static void writeMethodInfo(MethodInfo mi,
                                         HashMap<String, Integer> uniqueStrings, StreamWriter outValueStream)
             throws Exception {
-        int pos = uniqueStrings.get(mi.name).intValue();
```

Comment:
These are supposed to be the UNBOXED values.  By default .get() returns a Java `Integer` class; and we want to make sure that we are using the unboxed value later in the app.   Please revert that code.

Reply:
It should be autounboxed by the JVM, and this is also a build time step. I believe this change is ok.